README.md

```diff
@@ -1,2 +1,176 @@
 # Smart-Bike-Helmet
-##Abstract
+
+## Abstract
+
+Smart Bike Helmet is a protective headgear that provides enhanced protection to the rider with advanced features such as alcohol detection, accident detection, and live location tracking. It protects the rider by not allowing the bike’s ignition to turn ON if he/she is drunk or not wearing the helmet. This is done using an RF module. Its main purpose is to detect possible accidents and notify the rider's family about the same with the current location so that help could be provided timely.
+
+## Motivation
+
+Our inspiration was to learn the basics of IoT, Arduino, to work with sensors and to familiarize ourselves with the various platforms that are used for IoT through this project.
+<br>
+The aim of this project is to ensure safety of the rider and prevent deaths and injuries caused by road accidents. If such a product is used on a large scale then the number of road accident related deaths and cases would be reduced. It is because it would be compulsory to wear the helmet and not be drunk in order to ride the bike. Also in case an accident occurs, the family gets notified about it along with the current loccation of the driver. Through this we want to solve two issues : reducing the number of road accidents by reducing drink and drive cases and provision of timely treatment that could otherwise prove to be fatal.
+
+## Mechanical Aspect of the Design
+
+The main body is a basic helmet to which various sensors are attached that gives it its functionality. An alcohol sensor has been fitted inside the helmet right in front of the mouth of the rider. Force sensitive resistor has been placed on the top so that it can sense whether the helmet is being worn or not. The rest of the sensors and microcontroller are placed inside a plastic box and secured at the back of the helmet.
+
+## Electronic Aspect of the Design
+
+### The sensors used are as follows:
+
+### ADXL345 Accelerometer
+
+The ADXL345 is a low-power, 3-axis MEMS accelerometer modules with both I2C and SPI interfaces.The sensor consists of a micro-machined structure on a silicon wafer. The structure is suspended by polysilicon springs which allow it to deflect smoothly in any direction when subject to acceleration in the X, Y and/or Z axis. Deflection causes a change in capacitance between fixed plates and plates attached to the suspended structure. This change in capacitance on each axis is converted to an output voltage proportional to the acceleration on that axis.
+
+<img src="./acc.PNG" alt="acc">
```

Comment:
add images and videos in paragraph tags. for e.g [link](https://github.com/marsiitr/IoT-ChatBot/blob/7ad401e53fd94fa639e42c0710a70af0a1693533/README.md?plain=1#L5)

Reply:
not sure if I did this right.
please check

Reply:
it should be in <p> </p> tags, so that images can be aligned to center. you can take reference from previous year projects documentation.